docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -3,13 +3,73 @@
 Some examples for the development kits from Nordic Semiconductor support
 over-the-air Device Firmware Upgrade.
 
-Currently, the Bluetooth LE is the only available transport for performing the
-DFU operation and it uses nRF Connect SDK's
-[Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt).
-The upgrade can be done either using a smartphone application or a PC command
-line tool.
+Depending on the example, the DFU can be performed using the Matter-compliant
+OTA feature in which the Matter operational network is used for downloading a
+new firmware image, or using
+[Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt)
+over Bluetooth LE. In the latter case, the DFU can be done either using a
```

Comment:
How about expressing it more clearly that this is Nordic proprietary way?

Reply:
But I think it's not Nordic proprietary, right? SMP is just another protocol. I can probably only mention that it's not Matter-defined.